includes/class-newspack-newsletters-layouts.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Newspack Newsletter Layouts
+ *
+ * @package Newspack
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Main Newspack Newsletters Class.
+ */
+final class Newspack_Newsletters_Layouts {
+	/**
+	 * CPT for Newsletter layouts.
+	 * Name if funky because of 20 character restriction.
```

Comment:
```suggestion
	 * Name is funky because of 20 character restriction.
```

Reply:
Fixed in c35826a7f6e6264189d2d1801e01c78ec2e699f9